### ISSUE TITLE
sqlite_schema is missing

### ISSUE THREAD

**Issue by zhiqiangxu** [mattn/go-sqlite3#1003]
This query is supposed to return all tables:

```sql
SELECT * from sqlite_schema where type ='table' AND name NOT LIKE 'sqlite_%'
```

But an error is returned instead:

```
no such table: sqlite_schema
```

**Comment by rittneje** [comment]
What version of this library are you using?

**Comment by zhiqiangxu** [comment]
> What version of this library are you using?

v1.11.0

**Comment by rittneje** [comment]
The `sqlite_schema` table alias was introduced in SQLite 3.33.0, which was included with version 1.14.1 of this library. With older versions you will need to continue using the original name - `sqlite_master`.

**Comment by zhiqiangxu** [comment]
> The `sqlite_schema` table alias was introduced in SQLite 3.33.0, which was included with version 1.14.1 of this library. With older versions you will need to continue using the original name - `sqlite_master`.

upgrading to 1.14.1 fixed this issue, thanks!
